### ISSUE TITLE
how to compile the software??

### ISSUE THREAD

**Issue by christophered** [tesseract4java/tesseract4java#1]
Can You explain the steps to compile the tesseract-gui

**Comment by pedromadeira** [comment]
I also wanna know. It seems the best tesseract gui out there.

**Comment by pvorb** [comment]
Thank you for your interest in tesseract-gui.

In order to build the software, you need to clone both tesseract4java/tesseract-gui as well as tesseract4java/tesseract-tools and put the required third party libraries in the libs subdirectory of each of the two modules.

I understand that this is not straight forward, so I'll try to simplify the building process during the next couple of days.

**Comment by pvorb** [comment]
Here is a listing of the required dependencies:

#### tesseract-gui

~~~
$ ls -R lib/
lib/:
darwin_universal/  linux_x64/  linux_x86/  silk-icons.jar  win32/  win64/

lib/darwin_universal:

lib/linux_x64:
libliblept170.so  libtesseract.so

lib/linux_x86:

lib/win32:
liblept170.dll*  tesseract.dll*

lib/win64:
liblept170.dll*  tesseract.dll*
~~~

#### tesseract-tools

~~~
$ ls -R lib/
lib/:
ij.jar  ijblob-1.3.jar
~~~

**Comment by pvorb** [comment]
You can track progress on the branch [feature/integrate-tools](https://github.com/tesseract4java/tesseract-gui/tree/feature/integrate-tools).

**Comment by pedromadeira** [comment]
Thanks man. I'll wait for the simplified building process. Take your time!

**Comment by pvorb** [comment]
I realized I'll need to invest more work. See [this milestone](https://github.com/tesseract4java/tesseract-gui/milestone/1) for more info.

**Comment by christophered** [comment]
Can't you share a ready-made compiled executable of Tesseract GUI?

**Comment by pvorb** [comment]
@christophered A compiled executable is the goal, but there's a lot of work to do until I'm able to provide one.

The first task is include all information for building the software. Then I'll reduce the manual steps until I'm able to do automated builds, so I can provide an execution ready distributable.

I don't plan to make a video. Why would you want it?

**Comment by pvorb** [comment]
@christophered @pedromadeira I added a section to the readme with information on [how to build the software](https://github.com/tesseract4java/tesseract4java/blob/63139195b7f2e54b66a4ae162a12905fb17d3779/README.md#building-and-running-the-software).

What do you think? Can you reproduce a working build with these instructions?

**Comment by pvorb** [comment]
The migration from jtesseract to javacpp-presets/tesseract still leads to occasional crashes. I'll fix as much as I can before 0.1 will be released.

**Comment by pvorb** [comment]
@christophered @pedromadeira I released [Version 0.1.0](https://github.com/tesseract4java/tesseract4java/releases/tag/v0.1.0).
